Source/CocoaMQTTTypes.swift

```diff
@@ -59,12 +59,15 @@ extension UInt8 {
 }
 
 public enum CocoaMQTTError: Error {
+
+    @available(OSX 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+    enum FoundationConnection : Error {
```

Comment:
nit: Looking at the style followed in the codebase. Should we drop the space before the colon?

Reply:
Sure, I will do that :)